### ISSUE TITLE
create_directories failing due to symlink

### ISSUE THREAD

**Issue by Julusian** [factoriotools/factorio-docker#311]
I am trying to use the graftorio plugin which outputs some data inside the script-output folder.
This is failing with 
```
 783.141 Warning LuaGameScript.cpp:2105: write_file error: filesystem error: create_directories "/opt/factorio/script-output" failed: Not a directory [/opt/factorio/script-output]
```

This looks to be due to the symlinks https://forums.factorio.com/viewtopic.php?f=58&t=84510&p=492586&hilit=create_directories#p492586

I have tried to override the symlink by adding an additional mount to the same path, but the mount ends up mounting at /factorio/script-output instead of /opt/factorio/script-output, so is not of any use.

Do you have any suggestions on how to work around this?
Perhaps the symlinks could be created by /docker-entrypoint.sh, if the folders dont already exist?

**Comment by SuperSandro2000** [comment]
> Do you have any suggestions on how to work around this?

Convince the factorio devs to not handle symlinks like 3rd party citizens. 

> Perhaps the symlinks could be created by /docker-entrypoint.sh, if the folders dont already exist?

Data is expected to be on the volume which then wouldn't be the case anymore.

I will change the data directory in the factorio config to ``/factorio`` to resolve this issue. I did some quick tests and I hope there are no regressions from this.

If someones finds a regression please let me know asap and I try to find a solution or revert this.

**Comment by SuperSandro2000** [comment]
This will be fixed in 0.18.24-1.
